application/frontend/controllers/SiteController.php

```diff
@@ -77,8 +86,6 @@ public function actionStatus()
             ]);
             throw new Http500('Email Service problem.', $e->getCode());
         }
-
```

Comment:
Why did you remove this block from the function? Is the function even called anymore?

Reply:
I'm not sure what you're referring to.  The git diff makes things look funny, because it shows the `statusCode = 204` as having been added above and removed down here, where as to the human eye it's still in the same place it was before inside its original function.

Reply:
Nevermind. I figured it out. Exactly.